roles/keystone-setup/tasks/main.yml

```diff
@@ -42,6 +42,7 @@
                  tenant={{ item.tenant }}
                  token={{ admin_token }}
   with_items: keystone.users
+  when: "{{ item.name }}.enabled is undefined or {{item.name}}.enabled|bool"
```

Comment:
We can make that change... However, I used the syntax above since it was done that way in openstack-setup task  ursula/roles/openstack-setup/tasks/main.yml

- include: users.yml
  when: openstack_setup.add_users|bool or openstack_setup.add_users is undefined

- include: images.yml
  when: openstack_setup.add_images|bool or openstack_setup.add_images is undefined